docs/using-airbyte/oauth.md

```diff
@@ -0,0 +1,70 @@
+---
+products: all
+---
+
+# Using OAuth to Connect
+
+Many Airbyte connectors support OAuth 2.0, enabling secure and seamless integration between Airbyte and third-party APIs. This guide explains how OAuth works for connectors in Airbyte.
+
+## What is OAuth?
+
+[OAuth 2.0](https://oauth.net/2/) is an open standard for access delegation. It allows websites or applications to request limited access to your information on other platforms without exposing passwords. By using token-based authentication, OAuth enhances security and improves your user experience.
+
+## OAuth Flow in Airbyte
+
+The complexity of setting up OAuth flows is handled internally by Airbyte. When you set up a new connector, you only need to complete a few steps to use OAuth. Airbyte’s OAuth implementation involves three key components:
+
+1. **You**: The individual authorizing access to your account on a third-party platform.
+
+2. **Airbyte**: The application requesting authorization on your behalf.
+
+3. **The third-party API**: The service to which the connector needs access.
+
+This simplified diagram visualizes the connection process.
+
+```mermaid
+graph TD;
+    A[You] -->| Authorize Airbyte| B[Third-party authorization server]
+    B -->| Issues authorization code| C[Airbyte]
+    C -->| Exchanges authorization code for access token| B
+    C -->| Sends API Request with access token| D[Third-party resource server]
+    D -->| Returns protected data| C
+```
+
+## OAuth in Airbyte Cloud
+
+In Airbyte Cloud, all client credentials are securely managed by Airbyte. You only need to click the authentication button and use the third-party platform's UI to authorize Airbyte.
```

Comment:
❗ this is not true. Only some oauth applications are provided by airbyte. Specifically those connectors that airbyte maintains itself (github, google). For the rest users will need to bring their own oauth application